<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sorting for Machine Learning</string>
  </property>
  <widget class="QWidget" name="graphSpace" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>741</width>
     <height>370</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Based on the distribution, suggestions for:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Inflection Point2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Inflection Point1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>151</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string>Sort</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inflectionPoint1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="inflectionPoint2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="inflectionPoint1Top">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>59</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="inflectionPoint2Top">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>75</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Based on the distribution, suggested ranges for:       </string>
   </property>
  </widget>
  <widget class="QLabel" name="fileNameLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="inflectionPoint2Bottom">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>87</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="inflectionPoint1Bottom">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>86</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>61</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Upper</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>88</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Lower</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
